mftf/2.3/test/actions.md

```diff
@@ -2458,8 +2463,8 @@ See [waitForJS docs on codeception.com](http://codeception.com/docs/modules/WebD
 
 Attribute|Type|Use|Description
 ---|---|---|---
-`function`|string|optional|
-`time`|string|optional|
+`function`|string|optional| The function to be run after all javascript finishes.
```

Comment:
Not accurate. It actually executes the JavaScript and waits up to $timeout seconds for it to return true.